Exercise06_Complete

```diff
@@ -0,0 +1,49 @@
+#To see history of edits please refer to the python scripts labelled Exercise06_Part1 and Exercis06_Part2(misspelled when naming the file)
+
+#For Part 1
+
+import pandas
+import matplotlib.pyplot as plt
+
+#create new dataframe of the raw data
+bball= pandas.read_table('UWvMSU_1-22-13.txt',delim_whitespace=True)
+
+#setting running cumulative scores for both teams
+UWscore = [0]
+MSUscore = [0]
+time = [0]
+
+#loop to match scores with teams
+for i in range(0, bball.shape[0], 1):
+    if bball.team[i] == 'UW':
+        UWscore.append(UWscore[-1]+bball.score[i])
+        MSUscore.append(MSUscore[-1])
+    elif bball.team[i] == 'MSU':
+        MSUscore.append(MSUscore[-1]+bball.score[i])
+        UWscore.append(UWscore[-1])
+    time.append(bball.time[i])
+
+#Line plot of both teams where 'r-' is a red line for UW and 'g-' is a green line for MSU
+plt.plot(time, UWscore, 'r-', time, MSUscore, 'g-')
+
+#For Part 2
+
+import random
+guessesTaken=0
+number= random.randint(1,100)
+print('I am thinking of a number between 1 and 100')
+
+while guessesTaken <10:
+    print('Guess')
+    guess= input()
+    guess= int(guess)
+    
+    guessesTaken= guessesTaken + 1
+    
+    if guess < number:
+        print ('Higher')
+    if guess > number:
+        print ('Lower')
+    if guess == number:
+        break
+print('Correct!')
```

Comment:
You may want to put "print('Correct')" within the "if guess == number" so it won't print correct when you reach the maximum guesses.